content/cucumber/step-argument-transforms.md

```diff
@@ -85,101 +89,107 @@ end
 
 ### The Whole Table
 
-A table like this one may occur again in a creation step and again in a validation step. Using a Transform will keep you DRY. Transforms can also be used with tables. A table transform is matched via a comma-delimited list of the column headers prefixed with 'table:'
+Transforms can also be used with tables. Just as they do for Step Definitions, using a Transform with tables will keep your code DRY.
+
+A table like this may occur again in a creation Step, and again in a validation Step. 
+
+A table Transform is matched via a comma-delimited list of the column headers prefixed with `table:`:
 
 ```ruby
 
-1. features/step_definitions/user_steps.rb
+# 1. features/step_definitions/user_steps.rb
 
 Transform /^table:user,followers$/ do |table|
-table.map_headers! {|header| header.downcase.to_sym }
-table.map_column!(:user) {|user| User.find_by_name(user) }
-table.map_column!(:followers) {|count| count.to_i }
-table
+    table.map_headers! {|header| header.downcase.to_sym }
+    table.map_column!(:user) {|user| User.find_by_name(user) }
+    table.map_column!(:followers) {|count| count.to_i }
+    table
 end
 
 Given /^the following users$/ do |table|
-table.hashes.each do |row|
-row\[:user].followers = row\[:followers]
-end
+    table.hashes.each do |row|
+        row[:user].followers = row[:followers]
+    end
 end
 
 Then /^I expect the following users$/ do |table|
-table.hashes.each do |row|
-row\[:user].should_not be_nil
-row\[:user].followers.should == row\[:followers]
-end
+    table.hashes.each do |row|
+        row[:user].should_not be_nil
+        row[:user].followers.should == row[:followers]
+    end
 end
 ```
 
 ## Transform Wisdom
 
-Transforms are powerful and it is important to take care how you implement them. A mistake with them can often introduce strange and unexpected behavior.
+Transforms are powerful, and it is important to implement them with care. A mistake can often introduce strange and unexpected behavior.
 
-**1. Transforms always require that you specify a block variable even if you do not specify any groups to capture.**
+**1. Transforms always require that you specify a block variable, _even if you do not specify any groups to capture_.**
 
-With no capture groups specified the entire value is returned as a string to the block. This is contrast to how step definitions perform and can often trip you up. Cucumber will fail execution with a warning when you fail to specify at least one variable.
+With no capture groups specified, the entire value is returned as a string to the block. This is in contrast to how Step Definitions behave, and can be confusing. 
 
-```ruby
+Cucumber will fail execution with a warning when you fail to specify at least one variable.
```

Comment:
"+Cucumber will fail execution " slightly unclear. Would exit with a warning be better?